### ISSUE TITLE
Text content lacks 4.5:1 contrast ratio

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#573]
<dl><dt>Description</dt><dd>The contrast ratio between text and its background is not at least 4.5:1.
In Light theme. Text written inside the circle icons are 1 , 2 , 3 is having same color as background due to which user can not perceive them visually. </dd><dt>Impact</dt><dd>Serious</dd><dt>Recommendation</dt><dd>RULE : 
Small text (under 18 point regular font or 14 point bold font) MUST have a contrast ratio of at least 4.5 to 1 with the background.

HOW TO FIX: 
Fix this issue by adjusting the text and/or background to increase the contrast to at least 4.5 to 1.

REFERENCE: 
Deque University: https://dequeuniversity.com/class/visual-design2/contrast/text-against-background

BACKGROUND: 
People who have low vision or are colorblind may have difficulty reading text if the contrast between the text its background is insufficient. When the contrast ratio between text and its background is adequate, people who have low vision or are colorblind are more likely to be able to read the text.</dd><dt>Source code</dt><dd><pre><code><ol class="progress"><li><button type="button" aria-disabled="true" class="TooltipTabstop" aria-describedby="tooltip1">1</button></li><li><button type="button" aria-disabled="true" class="TooltipTabstop" aria-describedby="tooltip2">2</button></li><li><button type="button" aria-disabled="true" class="TooltipTabstop" aria-describedby="tooltip3">3</button></li></ol></code></pre></dd><dt>Checkpoint</dt><dd>Color Contrast (regular text) (1.4.3.a)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/components/TooltipTabstop</dd></dl>

**Comment by schne324** [comment]
duplicate
